resources/definitions/fdmprinter.def.json

```diff
@@ -5340,6 +5384,70 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": false,
                     "settable_per_meshgroup": false
+                },
+                "ironing_enabled":
+                {
+                    "label": "Enable Ironing",
+                    "description": "Go over the top surface one additional time, but without extruding material. This is meant to melt the plastic on top further, creating a smoother surface.",
```

Comment:
There is a flow setting, so it is not `without extruding material`

It doesn't really make sense to say that melting causes the top surface to be more flat.

Maybe something like `Smooth out bumps and crevices on flat top parts by passing over the top skin lines again after they have been printed.`